### DISCUSSION TITLE
Typo in documentation?

### DISCUSSION THREAD

**Discussion by mandras73** [pytest-dev/pytest#12948] [General]
Hi, take a look on these two code blocks
https://docs.pytest.org/en/latest/how-to/cache.html#the-new-config-cache-object

"If you run this command for the first time, you can see the print statement:"
and 
"
If you run it a second time, the value will be retrieved from the cache and nothing will be printed:"

For me, both look the same....
Cheers

**Comment by The-Compiler** [comment]
The difference in the output is exactly what's explained in the text. The first one shows the print output:

```
-------------------------- Captured stdout setup ---------------------------
running expensive computation...
```

while the second one does not.
